### PR TITLE
FAQ: missing i386 architecture

### PR DESCRIPTION
**Pull request by advayDev1**

### CODE REVIEW

**Comment by advayDev1** [comment]
@brunobowden ptal.

cc @maxbritto

**Comment by maxbritto** [comment]
Nice :)

File: FAQ.md [hunk 1]
```diff
@@ -506,3 +507,23 @@ local.properties (the Mac OS X developers should not use this):
...
+If you see a message similar to:
+```
+ld: warning: ignoring file /PATH/j2objcOutputs/lib/iosDebug/libPROJECT-j2objc.a,
+missing required architecture i386 in file /PATH/j2objcOutputs/lib/iosDebug/libPROJECT-j2objc.a
+(3 slices)
+```
+and additionally get linker errors, you are not building all the neccessary architectures.
+
+By default (for performance), we build only modern iOS device and simulator architectures.
+If you need i386 for older simulators, add the following to your build.gradle file:
```
Comment (brunobowden) [inline FAQ.md]: Make it clearer:

    older simulators (iPhone 5, 5c and earlier devices), add the following...

**Comment by brunobowden** [comment]
LGTM

File: FAQ.md (hunk 1 above)
Comment (advayDev1) [inline FAQ.md]: done

**Pull request merged**